Source/WebCore/rendering/RenderBox.h

```diff
@@ -705,6 +706,7 @@ class RenderBox : public RenderBoxModelObject {
     }
 
     bool establishesIndependentFormattingContext() const override;
+    bool createsBlockFormattingContext() const;
```

Comment:
Any reason why you chose "creates" instead of the spec term of "establishes" (same as the function above)?

Reply:
Nope. Changed.